common/lib/capa/capa/responsetypes.py

```diff
@@ -684,6 +687,7 @@ class ChoiceResponse(LoncapaResponse):
 
     """
 
+    human_name = _('Checkboxes')
```

Comment:
@cahrens I don't think so, as it's not one-to-one relationship between problem categories and response types. E.g. "Custom Javascript/Python" and "Drag and Drop" problem templates are all processed by `CustomResponse`.